Automattic/wp-calypso
# Added verify email page on WooCommerce landing page

#### Proposed Changes

If the user reaches Calypso and clicks on WooCommerce, but has not yet verified their account email, we should modify the information displayed to them to prompt them to confirm their email before they progress.



#### Testing Instructions



* Create a new site `/new` using a new account.
* After the site is created, go to the WooCommerce menu
* You should see the new page
<img width="937" alt="image" src="https://user-images.githubusercontent.com/13437011/171038462-fd87473d-15bf-4a06-89dd-3445689fb00f.png">

* Click on 'Edit email address'. It should redirect you to the profile settings.
* Go back
* Click on 'Resend verification email'
* Check if you've received the email
* Verify the email
* Refresh the page
* You should see the regular page
 
![image](https://user-images.githubusercontent.com/3801502/172607174-a077dfe0-3483-4653-9914-52f9b6975faf.png)




Closes #64165
